src/main/java/teammates/common/util/Const.java

```diff
@@ -61,7 +60,6 @@
     
     public static final String DEFAULT_SECTION = "None";
     
-    public static final String EVAL_PREFIX_FOR_INSTRUCTOR_PRIVILEGES = "eval%";
```

Comment:
Not used anymore

Added in 
~~~
commit 9343caa031ea6944929f8a0e52be96733a1ce03c
Author: franklingu <franklingujunchao@gmail.com>
Date:   Fri Jun 27 18:54:59 2014 +0800

    UI is roughly done
~~~
Made obselete in
~~~
commit 70681d7825e771b7bd1b6431c318211799332758
Author: Kang HJ <kanghongjin@gmail.com>
Date:   Sun May 10 19:50:12 2015 +0800

    Update more jsp files and Page Data
~~~